Wikipedia/Code/ArticleLocationCollectionViewController.swift

```diff
@@ -79,32 +74,22 @@ class ArticleLocationCollectionViewController: ColumnarCollectionViewController,
     // MARK: ArticlePreviewingDelegate
     
     override func shareArticlePreviewActionSelected(with articleController: ArticleViewController, shareActivityController: UIActivityViewController) {
-        guard let context = feedFunnelContext else {
-            super.shareArticlePreviewActionSelected(with: articleController, shareActivityController: shareActivityController)
-            return
-        }
         super.shareArticlePreviewActionSelected(with: articleController, shareActivityController: shareActivityController)
-        FeedFunnel.shared.logFeedDetailShareTapped(for: context, index: previewedIndexPath?.item, midnightUTCDate: context.midnightUTCDate)
     }
 
     override func readMoreArticlePreviewActionSelected(with articleController: ArticleViewController) {
```

Comment:
No need to override at all anymore if you're only calling `super`, you can remove this method.